.gitlab-ci/packet.yml

```diff
@@ -24,7 +24,7 @@ packet_centos7-flannel-addons:
 # ### MANUAL JOBS
 
 packet_centos7-weave-kubeadm-sep:
-  stage: deploy-part2
+  stage: unit-tests
```

Comment:
Note: once this PR is ready to merge, we need to revert back to `stage: deploy-part2`